src/custom/utils/trade.ts

```diff
@@ -70,6 +72,7 @@ export async function signAndPostOrder(params: PostOrderParams): Promise<AddUnse
     signer,
     recipient,
     allowsOffchainSigning,
```

Comment:
Do we still need this? (im only `isSmartContractWallet` is needed. We need to remove it from all the parameters)

Reply:
I am not sure what was the original intent of allowsOffchainSigning here, so I did not want to reuse this prop. Instead I wanted something more explicit (considering that isSmartContractWallet = potentially 1271)